bin/dotStartup.sh

```diff
@@ -124,6 +124,61 @@ your dotCMS application."
         fi
 fi
 
+# Code for bringing Open Distro up if needed/specified
+OPEN_DISTRO_HOST=https://127.0.0.1
+OPEN_DISTRO_PORT=9200
+OPEN_DISTRO_ALREADY_RUNNING=false
+OPEN_DISTRO_USER=admin
+OPEN_DISTRO_PASSWORD=admin
+
+health_check="$(curl "$OPEN_DISTRO_HOST:$OPEN_DISTRO_PORT/_cat/health?h=status" -u $OPEN_DISTRO_USER:$OPEN_DISTRO_PASSWORD --insecure)"
+if [ "$health_check" = 'yellow' ] || [ "$health_check" = 'green' ]; then
+  OPEN_DISTRO_ALREADY_RUNNING=true;
+fi
+
+echo "Is Open Distro already running? = $OPEN_DISTRO_ALREADY_RUNNING"
+
+if [ "$OPEN_DISTRO_ALREADY_RUNNING" = false ] ; then
+
+    ## check for open-distro arguments 
+    LAUNCH_OPEN_DISTRO=true
+
+    # idiomatic parameter and option handling in sh
+    while test $# -gt 0
+    do
+        case "$1" in
+            --skipOpendistro) 
+              echo "option 1"
+              LAUNCH_OPEN_DISTRO=false
+                ;;
+            --*) echo "bad option $1"
+                ;;
+        esac
+        shift
+    done
+
+    echo "Launching Open Distro? = $LAUNCH_OPEN_DISTRO"
+
+    if [ "$LAUNCH_OPEN_DISTRO" = true ] ; then
+        ## Bring up Open Distro
+        docker run -d --name dot_opendistro -e PROVIDER_ELASTICSEARCH_HEAP_SIZE=1500m -e PROVIDER_ELASTICSEARCH_DNSNAMES=elasticsearch -e ES_ADMIN_PASSWORD=$OPEN_DISTRO_PASSWORD -e discovery.type=single-node -p $OPEN_DISTRO_PORT:9200 gcr.io/cicd-246518/es-open-distro:1.2.0
```

Comment:
if this fails, we should send a message saying something like "Unable to start the Elasticseach docker container, please make sure you either start Elasticseach before you start dotCMS or have docker available so dotCMS can start the Elasticseach docker container"

Reply:
Done